main.go

```diff
@@ -1001,7 +994,7 @@ func collectNames(files []*ast.File) map[string]struct{} {
 }
 
 // transformGo garbles the provided Go syntax node.
-func transformGo(file *ast.File, info *types.Info, blacklist map[types.Object]struct{}, privateNameMap map[string]string, pkgPath string, existingNames map[string]struct{}, packageCounter *int) *ast.File {
+func transformGo(file *ast.File, info *types.Info, blacklist map[types.Object]struct{}, pkgScope *types.Scope, privateNameMap map[string]string, pkgPath string, existingNames map[string]struct{}, packageCounter *int) *ast.File {
```

Comment:
`pkgScope` should be renamed if it is different from `pkg.Scope()` in line 1058, if it is the same remove this parameter and use `pkg.Scope()` instead

Reply:
It is different, `pkgScope` is the scope of the package of the file that is getting transformed. `pkg.Scope` is the package scope of the identifier that is currently getting inspected. Yeah, the naming is a bit confusing

Reply:
What do you think `pkgScope` should be renamed to?

Reply:
renamed to `filePkgScope`, as it is the package scope of the file `transformGo` is called with